charts/cla/templates/deployment.yaml

```diff
@@ -26,29 +26,33 @@ spec:
           env:
           - name: HOST
             value: {{ .Values.claAssistant.host | default "example.com" }}
-          - name: GITHUB_CLIENT
-            value: {{ .Values.github.client | default "CLIENT" }}
-          - name: GITHUB_USER
-            value: {{ .Values.github.user | default "USER" }}
           - name: PROTOCOL
-            value: http
-          - name: GITHUB_ADMIN_USERS
-            value: {{ .Values.github.user | default "GITHUB_ADMIN_USERS" }}
+            value: https
```

Comment:
You should use `.Values.claAssisttant.protocol` here.

Reply:
ok